sympy/printing/mathml.py

```diff
@@ -73,6 +74,9 @@ def doprint(self, expr):
         res = xmlbstr.decode()
         return res
 
+    def _split_super_sub(self, name):
+        return (name, [], []) if self._settings["disable_split_super_sub"] else split_super_sub(name)
```

Comment:
I often see this style of writing a very long line with ternary if/else. It is much clearer just to use an ordinary if else:
```
if self._settings["disable_split_super_sub"]:
    return (name, [], []) 
else
    return split_super_sub(name)
```